src/cli.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { deploy } from "./js-api";
+
+const foundryToken = process.env.FOUNDRY_TOKEN;
+
+if (!foundryToken) {
+  throw Error(
+    `FOUNDRY_TOKEN environment variable is required to use single-spa-foundry-cli.`
+  );
+}
+
+yargs(hideBin(process.argv))
+  .command(
+    "deploy <microfrontendName>",
+    "deploy a microfrontend",
+    (yargs) => {
+      return yargs.positional("microfrontendName", {
+        describe: "The name of the microfrontend you wish to deploy",
```

Comment:
In my understanding of yargs, the describe option is a type of help text.

Reply:
yes that's correct